style.css

```diff
@@ -112,3 +112,9 @@ a {
 a:hover {
   opacity: 0.7;
 }
+
```

Comment:
This solution looks great. I'll keep it in mind. maybe we might need to use `floats` in some rare occasions.
So I won't close the **PR**.